torchtune/models/phi3/_sentencepiece.py

```diff
@@ -110,7 +110,13 @@ def decode(self, ids: List[int]) -> str:
         Returns:
             str: The decoded text.
         """
-        return self.spm_model.decode(ids)
+        ids_for_decode = []
```

Comment:
Why do we need this change? Does decode not handle special tokens correctly? Or whats going on here?

Reply:
Updated the PR description :)